src/edge/edgeManager.ts

```diff
@@ -798,6 +802,9 @@ export class EdgeManager {
                 description: Constants.STREAM_ANALYTICS_DESCRIPTION,
             },
             {
+                label: Constants.EVENT_GRID,

```

Comment:
seems alphabetically, event_gid should before machine learning?

Reply:
Updated